File: Java/PayPal.java
public class PayPal extends Payments{
    private String email;

    // CONSTRUCTOR
    public PayPal(String nombreTitular,String email){
        super(nombreTitular);
        this.email = email;
    }
    
    // GETTES AND SETTERS
    public String getEmail(){
        return email;
    }

    public void setEmail(String email){
        this.email = email;
    }

    
}

File: PHP/Paypal.php
<?php
    require_once('Payments.php');

    class Paypal extends Payments{

        public $email;

        public function __construct($nomTitular,$email) {
            parent::__construct($nomTitular); // asi se declara la SUPER CLASE
            $this->email = $email;
        }


    }

?>